### PR TITLE
new ui for learn more button

### PR DESCRIPTION
**Pull request by prince7703**
### Description

Please include a summary of the changes made in this PR. Make sure to explain **what** changes have been made and **why** they are necessary.

- **Related Issue**: #(Issue Number)

### Checklist

Before submitting your pull request, please ensure you have completed the following tasks:
- [ ] I have read the [Contributing Guidelines](./CONTRIBUTING.md) and followed the process outlined there.
- [ ] I have added tests that prove my fix or feature works as expected (if applicable).
- [ ] I have updated necessary documentation (if applicable).
- [ ] My changes are free from any merge conflicts.
- [ ] I have added my name and PR details to the `participants/participants.js` file (if contributing in Tier 1 or 2).

### PR Title Convention

Please ensure the PR title follows the required format:

**For Tier 1 Contribution**:

```
issue-<your-name>-<PR-number>
```
Example: `issue-johndoe-123`

**For Tier 2 Contribution**:

```
pr-add-<your-name>-<PR-number>
```

Example: `pr-add-johndoe-123`

### Screenshots (if applicable)

Please include screenshots or videos of your changes (especially if the changes are UI/UX related).

### Additional Information

Please provide any additional information that might be helpful for the maintainers or other contributors to understand your changes.

### CODE REVIEW

**Comment by iamAyushChamoli** [comment]
@prince7703 We'll need a darker gradient in the buttons because the current one (IMO) does not sit well with the current color scheme of the website.
Plus, unsure if this is a system issue or an actual bug but the hover effects are very choppy and laggy. Please optimize it anyway because we don't want the website to be choppy on low spec devices.

**Comment by prince7703** [comment]
okay

**Comment by iamAyushChamoli** [comment]
@prince7703 the color scheme is alright now but the hover effects are still choppy. Please optimize them to use less memory/resources.

**Comment by iamAyushChamoli** [comment]
@prince7703 I assume you're not understanding what I am trying to say here. Let me rephrase it; the color scheme and the styles that you used in the previous commit were all good. However, when I hover my cursor on the button, the hover effects are not smooth. They are lagging and slow. You don't have to change the color scheme, you have to update the logic of hover effects to make it smoother. If you are having trouble in that, you can get rid of this hover effect and use a different hover effect altogether.

**Comment by prince7703** [comment]
Behaviour is much more smooth than what it was in prev commit ,, also changed color gradient and tried a new as it looks good with the current background

**Comment by BlackPearl7703** [comment]
Check once i have changed the code one more time according to what you said
infact this time i have tried a new gradient which looks perfect for the
cards
